app/src/main/java/org/mifos/selfserviceapp/api/services/ClientService.java

```diff
@@ -16,15 +17,23 @@
  */
 public interface ClientService {
 
+    String CLIENT_ID = "clientId";
```

Comment:
I am assuming you abstracted this out as a constant, so this should be ```private static final```. What do you think ?